interfaces/MCube-Accel-MC36xx/src/user_drivers/platform/console/m_drv_console.c

```diff
@@ -39,8 +39,8 @@
 void mcube_printf(const char *str, ...)
 {
     /** Function hook by customer. */
-    uart2_write((uint8_t *)str, strlen(str), NULL); // send next string character
-    uart2_finish_transfers();
+    uart_write((uint8_t *)str, strlen(str), NULL); // send next string character
```

Comment:
This is a functionnal change. Why is it necesssary

Reply:
in uart module in SDK the declaration for the uart is :
void uart_write(uint8_t *bufptr, uint32_t size, void (*callback) (uint8_t));
also in config there is this comment :
/****************************************************************************************************************/
/*UART Console Print. Enables serial interface logging mechanism. If CFG_PRINTF is defined CFG_PRINTF_UART2     */
/* controls the uart module used. If it is defined UART2 is used. If not, UART is used. uart or uart2 driver    */
/* must be included in project respectively.                                                                    */
/****************************************************************************************************************/